woocommerce/woocommerce-ios
# [TTPUK] Change Try a Payment amount to 30p in the UK

Closes: #10889


## Description


In the UK, the minimum charge amount is £0.30. We use the minimum charge of $0.50 for Try a Payment in the US. Previously, this carried over as a £0.50 trial payment when that flow is followed in the UK, but it's better for our merchants if we use the minimum amount in the UK as well.

This PR updates the Try a Payment screen's description to include the correctly formatted amount, in the right currency, and uses the `minimumAllowedChargeAmount` from the card reader configuration to set the trial payment amount.

## Testing instructions


1. Using a UK WooPayments store
2. Navigate to `Menu > Payments > Try Out Tap to Pay on iPhone`
3. Tap through the Set up screens, if shown
4. Observe that the description says "Try taking a payment of £0.30 ..."
5. Tap `Try a Payment`
6. Observe that the try a payment flow uses £0.30 for its payment

Repeat the above test with a US-based store, and check that the amount is $0.50


## Screenshots



https://github.com/woocommerce/woocommerce-ios/assets/2472348/a95f83e7-e856-4aa3-94fa-39f0f11fceca


---
